citrination_client/views/data_view_builder.py

```diff
@@ -63,13 +63,13 @@ def add_raw_descriptor(self, descriptor):
         """
         self.configuration['descriptors'].append(descriptor)
 
-    def add_relation(self, inputs, outputs, relation_type='lolo', options=None):
+    def add_relation(self, inputs, output, relation_type='lolo', options=None):
```

Comment:
Do you run any checks to ensure the input(s) and output are in the dataview builder's list of descriptors

Reply:
Im not currently, but that is a good check.  Ill add it.